lib/facil/http/http.h

```diff
@@ -7,6 +7,12 @@ Feel free to copy, use and enjoy according to the license provided.
 */
 #define H_HTTP_H
 
+#ifdef __GNUC__
```

Comment:
What if `__GNCU_C` isn't defined?

Also...

Should this be here: https://github.com/boazsegev/facil.io/blob/master/lib/facil/fio.h#L221-L229

Reply:
Yes

Reply:
I would be happy if you could move this test to `fio.h` before I merge.

Either way - thank you very much for all the wonderful effort!

👏🏻👏🏻👏🏻👏🏻🙏🏻🙏🏻🙏🏻👍🏻👍🏻☺️